src/client/screens/RegisterScreen.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import {
+  KeyboardAvoidingView,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { RootStackParamList } from '../../../App';
+import { auth } from '../../server';
+import styles from '../styles/StyleSheet';
+import Account from '../../../firebase/account';
+
+type RegisterScreenProps = NativeStackScreenProps<
+  RootStackParamList,
+  'RegisterScreen'
+>;
+
+const RegisterScreen: React.FC<RegisterScreenProps> = (props) => {
+  const [name, setName] = useState('');
```

Comment:
Assuming you want these to be strings, you can pass in a type as so:
useState<string>('');